packages/cozy-stack-client/src/__tests__/CozyStackClient.spec.js

```diff
@@ -270,6 +284,60 @@ describe('CozyStackClient', () => {
       }
     })
   })
+
+  describe('refreshToken', () => {
+    beforeAll(() => {
+      global.fetch = require('jest-fetch-mock')
+    })
+
+    beforeEach(() => {
+      fetch.resetMocks()
+      fetch.mockResponse(FAKE_APP_HTML)
+    })
+
+    const getClient = () =>
+      new CozyStackClient({
+        uri: 'http://cozy.tools:8080//',
```

Comment:
Please note we recently had issues with cozy-desktop CI failing because of leading double slashes in URLs, not sure whether/why it actually works here :wink:

Reply:
Fixed anyways. Thx